drivers/dcf77/dcf77.c

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright (C) 2019 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+/**
+ * @ingroup     drivers_dcf77
+ * @{
+ *
+ * @file
+ * @brief       Device driver implementation for the dcf 77
+ *              longwave time signal and standard-frequency radio station
+ *
+ * @author      Michel Gerlach <michel.gerlach@haw-hamburg.de>
+ *
+ * @}
+ */
+
+#include <stdint.h>
+#include <string.h>
+
+#include "log.h"
+#include "assert.h"
+#include "xtimer.h"
+#include "periph/gpio.h"
+
+#include "dcf77.h"
+#include "dcf77_params.h"
+
+#define ENABLE_DEBUG    (0)
+#include "debug.h"
+
+/* Persistent level longer than 1200ms starts a new cycle  */
+#define DCF77_PULSE_START_HIGH_THRESHOLD_MICRO_SEC        (1200000U)    /*~1200ms*/
```

Comment:
Well you got to rename all the occurrences too, otherwise it won't compile :wink: 
Also no need to spell it out, `DCF77_PULSE_START_HIGH_THRESHOLD_MS` would be the more conventional way to name it.

Reply:
of course you have to this... sry

Reply:
Uh one more thing - it's 1200ms but the value of the define is actually `1200000` µs
So it should be `DCF77_PULSE_START_HIGH_THRESHOLD_US` because the value of the define is in µs, not ms.